js/navigation.jquery.js

```diff
@@ -304,6 +304,90 @@ class Navigation {
     // Handle in-page bookmarks.
     if (document.location.hash) this.hashChanged();
     $(window).on('hashchange', $.proxy(this.hashChanged, this));
+
+    // Accessibility
+    this.$container.on('keydown', (ev) => {
+      const $focus = $(document.activeElement);
+
+      const $li = $focus.parent();
+      const isEnterOrDown = ev.keyCode === 40 || ev.keyCode === 13;
```

Comment:
Can we extract these keycodes to a single place so it's clear which ones we're catching at a glance (enter/up/down/left/right I think)

Reply:
We could do that, or I could change to use <a href="https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key">`event.key`</a> - do you think browser support is good enough?

Reply:
Yes, I think support's good enough.